EKF/RingBuffer.h

```diff
@@ -79,26 +81,15 @@ class RingBuffer
 		_head = 0;
 		_tail = 0;
 
-		// set the time elements to zero so that bad data is not
-		// retrieved from the buffers
-		for (uint8_t index = 0; index < _size; index++) {
-			_buffer[index] = {};
-		}
-
 		_first_write = true;
 
 		return true;
 	}
 
-	void unallocate()
```

Comment:
This will probably make lots of noise for valgrind, even if it's ok in the real world.